ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -106,26 +107,46 @@
                                             update-type)]))))
 
 (defn- get-provider-concept-ids
```

Comment:
Rename this function to something like `get-provider-collection-concept-ids`.

Reply:
Done.